project/config/cvocni/config/block.block.sidemenu.yml

```diff
@@ -27,3 +28,12 @@ visibility:
     id: request_path
     negate: true
     pages: '<front>'
+  'entity_bundle:node':
+    id: 'entity_bundle:node'
+    negate: false
+    context_mapping:
+      node: '@node.node_route_context:node'
+    bundles:
+      basic_page: basic_page
+      news: news
```

Comment:
I don't think the sidemenu should be set for News or Publication pages.

Reply:
Thanks Adam. Done.